### PR TITLE
R and RStudio setup

### PR DESCRIPTION
**Pull request by stephan-koenig**
Hi Axel,

Would you please look over the R and RStudio setup information (`Rmds/Rmds/r_and_rstudio.Rmd`, you can ignore the other files) and see if it makes sense? Just pull the `r-setup` branch to your computer, build the bookdown and then looking at that chapter would do the trick.

Here is a short article on [how to do reviews of pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) in case you want to leave any comments.

I would like to publish the setup instructions by tomorrow so students have enough time to go through them.

Thanks,

Stephan

### CODE REVIEW

File: Rmds/r_and_rstudio.Rmd [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# R and RStudio {-}
+
+To complete the installs of R and RStudio, you must have administrator privileges on your machine. Choose the version appropriate for your operating system. You can also watch a video for a [Mac](https://youtu.be/Oy9BPxt8gQ0?t=270) or a [Windows](https://youtu.be/7f9ADV2v7CI?t=336) install (you only need to install R and RStudio, and none of the other programs mentioned in the videos). Please also note that the versions of R and RStudio in the video are older. Instead, install or update to the latest version supported by your operating system.
+
+1. R https://cran.r-project.org/
+    - version `r getRversion()` (only for macOS Yosemite or older: legacy version `r params$r_legacy_version`)
+    - Choose among "Download and Install R" section (**not** "Source Code for all Platforms").
+    
+        **Note for Mac users:** If you have an M1 (and higher) Mac, you **must** install the Intel version of R (Bioconductor packages installed below are not yet compatible with the arm64 version).
```
Comment (hauduc) [inline Rmds/r_and_rstudio.Rmd:9]: Change to "M1 or later"?

File: Rmds/r_and_rstudio.Rmd [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        ```
+          There is a binary version available, but the source version is later:
+        
+        <table of package names>
+        
+        Do you want to install from sources the package which need compilation? (Yes/no/cancel)
+        ```
+
+        Always enter `no` followed by <kbd>enter</kbd>.
```
Comment (hauduc) [inline Rmds/r_and_rstudio.Rmd:38]: Capital E on "enter"?

File: Rmds/r_and_rstudio.Rmd [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# R and RStudio {-}
+
+To complete the installs of R and RStudio, you must have administrator privileges on your machine. Choose the version appropriate for your operating system. You can also watch a video for a [Mac](https://youtu.be/Oy9BPxt8gQ0?t=270) or a [Windows](https://youtu.be/7f9ADV2v7CI?t=336) install (you only need to install R and RStudio, and none of the other programs mentioned in the videos). Please also note that the versions of R and RStudio in the video are older. Instead, install or update to the latest version supported by your operating system.
+
+1. R https://cran.r-project.org/
+    - version `r getRversion()` (only for macOS Yosemite or older: legacy version `r params$r_legacy_version`)
```
Comment (hauduc) [inline Rmds/r_and_rstudio.Rmd:6]: Should the sub-bullets be numbered too to show that the order matters?

**Review by hauduc — COMMENTED** [review]
Hi Stephan,

These setup instructions look good to me! I've suggested a few minor edits.

Best,
Axel

File: Rmds/r_and_rstudio.Rmd (hunk 1 above)
Comment (stephan-koenig) [reply Rmds/r_and_rstudio.Rmd:9]: Not sure what is more correct here, I went with the language they used on the R install page.

File: Rmds/r_and_rstudio.Rmd (hunk 2 above)
Comment (stephan-koenig) [reply Rmds/r_and_rstudio.Rmd:38]: I have used lowercase for all modifier/special keys elsewhere on the website. (I cannot decide what way I find better lower vs. uppercase and there does not seem to be a convention.)

File: Rmds/r_and_rstudio.Rmd (hunk 3 above)
Comment (stephan-koenig) [reply Rmds/r_and_rstudio.Rmd:6]: Ok, I will create subheaders and then use numbering instead of bullet points.

**Comment by stephan-koenig** [comment]
Thanks!

**Pull request merged**